### PR TITLE
[NetNow.co] Update and merge rulesets

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by Bisaloo** [comment]
I added a comment just before the rule so we don't forget about it in case we run the script to remove snapping redirects again.

File: src/chrome/content/rules/NetNow.co.xml [hunk 1]
```diff
@@ -1,50 +1,18 @@
...
-
-
 	Mixed content:
-
-		- css from $self *
-		- fonts from $self *
-		- Images from $self *
-
-	* Secured by us
+		www.netnow.co
```
Comment (J0WI) [inline src/chrome/content/rules/NetNow.co.xml]: I was not able to reproduce

Comment (J0WI) [inline src/chrome/content/rules/NetNow.co.xml]: See also `asi.netnow.co`

Comment (Bisaloo) [inline src/chrome/content/rules/NetNow.co.xml]: I confirm I have MCB issues on `www.netnow.co` on Chromium.

**Pull request merged**